asapdiscovery-data/asapdiscovery/data/fragalysis.py

```diff
@@ -9,41 +9,65 @@
 MPRO_API_CALL = {
     "target_name": "Mpro",
     "proteins": "",
-    "event_info": "false",
-    "sigmaa_info": "false",
-    "diff_info": "false",
-    "trans_matrix_info": "false",
-    "NAN": "false",
-    "mtz_info": "false",
-    "cif_info": "false",
-    "NAN2": "false",
-    "map_info": "false",
-    "single_sdf_file": "false",
-    "sdf_info": "true",
-    "pdb_info": "true",
-    "bound_info": "true",
-    "metadata_info": "true",
-    "smiles_info": "true",
-    "static_link": "false",
+    "event_info": False,
+    "sigmaa_info": False,
+    "diff_info": False,
+    "trans_matrix_info": False,
+    "NAN": False,
+    "mtz_info": False,
+    "cif_info": False,
+    "NAN2": False,
+    "map_info": False,
+    "single_sdf_file": True,
+    "sdf_info": True,
+    "pdb_info": False,
+    "bound_info": True,
+    "metadata_info": True,
+    "smiles_info": True,
+    "static_link": False,
+    "file_url": "",
+}
+
+MAC1_API_CALL = {
+    "target_name": "Mac1",
+    "proteins": "",
+    "event_info": False,
+    "sigmaa_info": False,
+    "diff_info": False,
+    "trans_matrix_info": False,
+    "NAN": False,
+    "mtz_info": False,
+    "cif_info": False,
+    "NAN2": False,
+    "map_info": False,
+    "single_sdf_file": True,
+    "sdf_info": True,
+    "pdb_info": False,
+    "bound_info": True,
+    "metadata_info": True,
+    "smiles_info": True,
+    "static_link": False,
```

Comment:
What is the difference between these two dictionaries? It seems like they share a lot of data and maybe what we want is not having a dictionary for every target but actually have some sort of sane defaults and change the dictionary as needed "on the fly", for example the `target_name` should change but probably many of the other options should stay the same.

Reply:
oh yeah that's a good point, they should be identical other than the `target_name` field. do you want to handle this in your changes or should I?

Reply:
I can take care of these changes